### PR TITLE
Update yaml description (diskspace)

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `diskspace`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

**Review by shyamvalsan — DISMISSED** [review]

File: collectors/diskspace.plugin/metadata.yaml [hunk 1]
```diff
@@ -26,7 +32,7 @@ overview:
     description: ''
   default_behavior:
     auto_detection:
-      description: ''
+      description: 'The plugin reads data from /proc/self/mountinfo and /proc/diskstats file.'
```
[the remark this answers is not in the document]
Comment (andrewm4894) [reply collectors/diskspace.plugin/metadata.yaml:35]: just make a commit suggestion - its literally less typing than a comment lol

Comment (andrewm4894) [inline collectors/diskspace.plugin/metadata.yaml:35]:
```suggestion
      description: 'The plugin reads data from `/proc/self/mountinfo` and `/proc/diskstats file`.'
```

[the remark this answers is not in the document]
Comment (andrewm4894) [reply collectors/diskspace.plugin/metadata.yaml:35]: (i took my own advice too :) )

File: collectors/diskspace.plugin/metadata.yaml [hunk 2]
```diff
@@ -36,14 +42,46 @@ setup:
     list: []
   configuration:
     file:
-      name: ''
-      description: ''
+      name: 'netdata.conf'
+      description: 'This is netdata main configuration file'
     options:
-      description: ''
+      description: 'Options are inside `[plugin:proc:diskspace]`, you can also specify per mount point `[plugin:proc:diskspace:mounpoint]`'
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/diskspace.plugin/metadata.yaml:48]: file.section_name should be `[plugin:proc:diskspace]`

[the remark this answers is not in the document]
Comment (thiagoftsm) [reply collectors/diskspace.plugin/metadata.yaml]: Thank you guys, I addressed the typo and suggestion

**Review by shyamvalsan — DISMISSED** [review]

**Review by andrewm4894 — DISMISSED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Review by sashwathn — APPROVED** [review]

**Comment by thiagoftsm** [comment]
Hello guys,

I fixed the wrong rebase.

**Review by sashwathn — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**